subgraph-js/src/directives.ts

```diff
@@ -173,7 +189,7 @@ export function typeIncludesDirective(
   directiveName: string,
 ): boolean {
   if (isInputObjectType(type)) return false;
-  const directives = gatherDirectives(type as GraphQLNamedTypeWithDirectives);
```

Comment:
We probably want to remove the special case for `isInputObjectType` just before this line? (I'll note that this method is only used internally in the case of object types, so doesn't matter so far for our internal usage, but since it's exported ...).